questions/README.md

```diff
@@ -9,6 +9,10 @@
 
     Answer ->
 
+## [For Python developers] Can you name 5 python librairies that you have you used for your personal or studies project and how they were useful? 
```

Comment:
@Dynnammo Typo here : "that you have used", not "that you have you used"

Reply:
Nice shot @MoretS , I fixed the typo and rewrite the question to be more understandable